scripts/install-chef-suse.sh

```diff
@@ -130,7 +131,7 @@ pipe_show_and_log () {
         rm -f $t
         dialog --clear >&3
     fi
-    tee -a /dev/fd/3 /dev/fd/4 > /dev/null
+    tee -a /dev/fd/4 >&3
```

Comment:
So this means this patch is tested with the new and the old installer code path of https://github.com/SUSE-Cloud/automation/pull/654